examples/visual-tests/amp-story/amp-story-tooltip-desktop.js

```diff
@@ -20,34 +20,40 @@ const {verifySelectorsVisible} = require('../../../build-system/tasks/visual-dif
 module.exports = {
   'tapping on a clickable anchor should show the tooltip': async (page, name) => {
     await page.tap('.next-container > button.i-amphtml-story-button-move');
-    await page.waitFor('amp-story-page#page-2[active]');
```

Comment:
Couldn't this still be flaky, if the navigation takes a long time?

I feel like maybe we should wait for page 2 to become active, _then_ wait however many milliseconds are necessary for the animation to complete?

Reply:
Sure, I re-added those.